conversions/binary_to_gray.py

```diff
@@ -0,0 +1,43 @@
+"""
+This code converts gray code to binary
+and
+binary code to gray code.
```

Comment:
What is gray code?!?
```suggestion
Convert gray code to binary and binary code to gray code.
```